src/components/gcds-header/gcds-header.tsx

```diff
@@ -0,0 +1,153 @@
+import { Component, Element, Host, Prop, h } from '@stencil/core';
+import { assignLanguage } from '../../utils/utils';
+
+const i18n = {
+  "en": {
+    "skip": "Skip to main content"
+  },
+  "fr": {
+    "skip": "Passer au contenu principal"
+  }
+}
+
+@Component({
+  tag: 'gcds-header',
+  styleUrl: 'gcds-header.css',
+  shadow: true,
+})
+export class GcdsHeader {
+
+  private lang: string;
+
+  @Element() el: HTMLElement;
+
+  /**
+  * GcdsLangToggle - The href attribute specifies the URL of the opposite language page
+  */
+  @Prop({ reflect: false, mutable: false }) langHref: string;
+
+  /**
+  * GcdsSignature - The variant of the Government of Canada signature
+  */
+  @Prop({ reflect: false, mutable: false }) signatureVariant: 'colour' | 'white';
+  /**
+  * GcdsSignature - GCDS signature links to Canada.ca
+  */
+  @Prop({ reflect: false, mutable: false }) signatureHasLink: boolean = true;
+  
+  /**
+  * Top navigation - Skip to content href
+  */
+  @Prop({ reflect: false, mutable: false }) skipToHref: string;
+
+  async componentWillLoad() {
+    // Define lang attribute
+    this.lang = assignLanguage(this.el);
+  }
+
+  private get renderTopNav() {
+    if (!!this.el.querySelector('[slot="topnav"]')) {
+      return <slot name="topnav"></slot>;
+    } else if(this.skipToHref) {
+      return (
+      <nav class="topnav">
+        <gcds-button
+          button-type="link"
+          button-role="skip-to-content"
+          href={this.skipToHref}
+        >
+          {i18n[this.lang].skip}
+        </gcds-button>
+      </nav>);
+    } else {
+      return;
+    }
+  }
+
+  private get renderToggle() {
+    if (!!this.el.querySelector('[slot="toggle"]')) {
+      return <slot name="toggle"></slot>;
+    } else if(this.langHref) {
+      return (
+        <section class="toggle">
```

Comment:
It feels weird to use a section tag for this. I know you copied this from what's on Canada.ca but maybe we can update that in the future.